app/views/summary/handovers.html.erb

```diff
@@ -0,0 +1,100 @@
+<% content_for :switcher do %>
+  <%= render '/layouts/prison_switcher' %>
+<% end %>
+
+<div class="govuk-!-margin-bottom-4">
+  <%= render "search/search_box" %>
+</div>
+
+<%= render(:partial => 'summary_subnav', :locals => {:active => :handovers}) %>
+
+<h1 class="govuk-heading-xl govuk-!-margin-bottom-4">Handover cases</h1>
+<p>
+  All cases that are within the handover period or start the handover process to the community in the next 30 days.
```

Comment:
Similar thought to above, I'll wait to hear if @lmoney-moj 's had the chance to look at these rather than rabbiting on at poor @ollietreend !